src/shared/lib/CopyNumberUtils.ts

```diff
@@ -0,0 +1,22 @@
+export const ALTERATIONS = {
```

Comment:
For the OncoKB API, there are only 2 valid values for CNA: `DELETION` and `AMPLIFICATION`. And here is the corresponding legacy code:

```JavaScript
switch(mapData.getValue(_id, "alter")) {
    case 2:
        alter = 'amplification';
        break;
    case -2:
        alter = 'deletion';
        break;
    default:
        alter = null;
}
```